### ISSUE TITLE
plugin:@graphql-eslint/all runs into error caused by incorrect "require-description" lint config

### ISSUE THREAD

**Issue by stephenkoo** [dotansimha/graphql-eslint#645]
**Describe the bug**

When running eslint using `plugin:@graphql-eslint/all`, get the error:

```
Error: .eslintrc.js#overrides[1] » plugin:@graphql-eslint/all:
        Configuration for rule "@graphql-eslint/require-description" is invalid:
        Value [] should NOT have fewer than 1 items.
```

This is even when disabling `"@graphql-eslint/require-description"`:
```js
    {
      files: ["*.graphql"],
      extends: ["plugin:@graphql-eslint/all"],
      rules: {
        "@graphql-eslint/require-description": "off",
      },
    },
```

**To Reproduce**

Create an eslint.js as above and run eslint.

**Expected behavior**

Lint to run without error.

**Environment:**

- OS: macOS 11.6
- `@graphql-eslint/...`: 2.2.0
- NodeJS: 16.9.1

**Additional context**

I believe this issue has a similar fix to the ones applied here:
- https://github.com/LD4P/sinopia_editor/issuec/1276
- https://github.com/airbnb/javascript/issues/2105

**Comment by JustinTRoss** [comment]
FYI - This issue is now reproducible in this repo 😅 :
https://github.com/JustinTRoss/graphql-eslint-issue-repro

**Comment by B2o5T** [comment]
Yes, I fixed this issue but only for next major version of graphql-eslint, that will be released very soon (we splitted `recommended` and `all` config into 4 configs - `schema-recommended`, `operations-recommended`, `schema-all` and `operations-all` configs)

`graphql-eslint` v3 is coming 🤟

**Comment by lughino** [comment]
Hi @B2o5T , is there an ETA for releasing v3?
I'm currently blocked from that as well.

**Comment by dotansimha** [comment]
> Hi @B2o5T , is there an ETA for releasing v3? I'm currently blocked from that as well.

We'll do it later this week! :)

**Comment by dotansimha** [comment]
Fixed in `@graphql-eslint/eslint-plugin@3.0.0`
